recipes-devtools/erlang/erlang-native_18.1.5.bb

```diff
@@ -5,6 +5,8 @@ inherit native
 
 PR = "r0"
 
+DEPENDS_append = " ncurses-native"
```

Comment:
I think you could replace adding this line to every `erlang-native-*` native recipe with the following line:
```
DEPENDS_append_class-native = " ncurses-native"
```
added to `erlang.inc`